src/deployment/deployment-hapi-plugin.ts

```diff
@@ -12,26 +12,21 @@ import DeploymentModule, {
   isRawDeploymentHostname,
 } from './deployment-module';
 
-import { gitlabHostInjectSymbol } from '../shared/gitlab-client';
 import { Logger, loggerInjectSymbol } from '../shared/logger';
 import memoizee = require('memoizee');
```

Comment:
As I was already working on the next set of changes for this file, I ended up restructuring and clarifying this class quite a bit. This is fixed in #176.